websockify/websocketproxy.py

```diff
@@ -245,13 +257,19 @@ def do_proxy(self, target):
                 # Receive target data, encode it and queue for client
                 buf = target.recv(self.buffer_size)
                 if len(buf) == 0:
+
+                    # Send queued target data to the client
+                    c_pend = self.send_frames(cqueue)
+                    cqueue = []
+
                     if self.verbose:
                         self.log_message("%s:%s: Target closed connection",
                                 self.server.target_host, self.server.target_port)
                     raise self.CClose(1000, "Target closed")
 
-                cqueue.append(buf)
-                self.print_traffic("{")
+                if len(buf) > 0:
+                    cqueue.append(buf)
+                    self.print_traffic("{")
```

Comment:
This change doesn't look meaningful. How can it reach this without `len(buf) > 0`?